okteto/okteto
# feat: add smart builds

## Description.
This PR pretends to add smart builds which means to manage each service in a repository individually instead of the current approach which is that the build related of a service is based on the repository commit. With this new approach, build context is check and only if the build context related to a service has changes then the image will be build again, but only for that service.

## How to test it
1. using okteto/movies with no changes
2. add a new file (`touch test.txt`) into `rent` folder.
3. run `okteto build` 
4. check images are build for the new generated hash
5. build again (`okteto build`)
6. check only `rent` service is built again because is the one with changes
